pegen/pegen.c

```diff
@@ -796,31 +783,13 @@ alias_for_star(Parser *p)
     return alias(str, NULL, p->arena);
 }
 
-/* Returns the first element of seq or previous if seq is empty */
-void *
-seq_get_head(void *previous, asdl_seq *seq)
```

Comment:
We should remove this two from the header file and the `helpers.md` file now that they are not used anymore.